package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "dialogflow",
+  "name": "@google-cloud/dialogflow",
   "description": "Dialogflow API client for Node.js",
   "version": "1.1.2",
```

Comment:
Since it will be a new package, should we start with v1.0.0?
Or, if we publish it now, just weeks before Node 8 deprecation, maybe make it v0.1.0? :)  WDYT @JustinBeckwith @bcoe?